src/assets/styles/custom-scroll.less

```diff
@@ -4,6 +4,8 @@
 
 .side-scroll {
   padding-bottom: 0px;
+  border-top: 1px solid @dark-background;
+  transition: border 0.7s;
```

Comment:
@WPFerg Could replace with the transition mixin - though not sure what it adds and this is more explicit that it's only the border being animated?

@owennw That selector doesn't match anything. I'll remove